### ISSUE TITLE
Version 2.0 prerelease available

### ISSUE THREAD

**Issue by soumak77** [base-apps/angular-base-apps#58]
I published the docs for the 2.0 prerelease to https://base-apps.github.io/angular-base-apps.    Give it a try and post any issues you find.

@tolyo @mattgrande @gakimball @AntJanus @Seb-

**Comment by Seb-** [comment]
Great job. My notes:
Let me know if I should make separate issues with them.
- [x] I don't like the fact that the currently released version does not have a doc online anymore. It would be nice to be able to host multiple versions of the doc at the same time. At least the current release and the one from master, and ideally every version for people who do not wish to upgrade immediately. I know it's possible do have them locally but the users don't want to and shouldn't be expected to do that.
- [x] I would consider a separate menu for the migration guide from 1.2 to 2. This is pretty big and shouldn't be missed. That page should be linked at several places, including where it is currently (Installation) but also on the Changelog page I believe.
- [ ] It would be pretty neat to provide a tool (probably best to be run with node) to help with the migration. Except the "foundation → base" most of other strings are easy to search and replace. I'm not thinking entirely automated tool, but probably one listing all the found occurrences and allowing the user to discard some and accept the rest, or to accept some and discard the rest, either by file, folder or individual occurrences. I know some commands or tools inside GUI editors can already do more or less all of that, but again best to expect the least from the users.
- [x] The CHANGELOG.md file has not been updated accordingly.
- [x] I have read somewhere there was a test made and a message shown if the current browser was not supported. This is excellent. It would be nice to include on the Compatibility page how to enable or disable this feature.
- [x] https://base-apps.github.io/angular-base-apps/#!/grid
  "The Grid in Action" > "View Templates" button leads to the homepage. Should be fixed to lead to the correct place or be removed (preferably fixed).
- [x] Links on Action Sheet, Menu Bar, Title Bar, Block List, Utility Classes still have the `href="#"` and lead to the Homepage. Same for the Search buttons on the Forms page (no href but same issue).
- [ ] This is not a doc issue but while I'm at it, I just noticed that the dark button doesn't change color on `:hover` or `:focus` unlike the others. And for some others (especially primary), the difference between the 2 states are really hard to notice. To be fair only the difference for the secondary is clear enough for my taste.
- [ ] Meter & Progress Elements should not be on the Form page at all, but on a separate page. Do meters still not support IE? Is it possible to make them support it? Is it a good idea to have individual elements not respecting what is announced on the Compatibility table? Probably not.
- [x] https://base-apps.github.io/angular-base-apps/#!/accordion
  "External Triggers"
  If 1 and 2 are closed, Toggle 3 button is annoying, because it moves place, which is really bad for a toggle button.
- [x] https://base-apps.github.io/angular-base-apps/#!/card
  Card missing the secondary and info examples.
- [x] https://base-apps.github.io/angular-base-apps/#!/labels-badges
  When the labels are going over 2 lines (ore more) they touch each other, it would be nice to have a vertical spacing between them (using a `p` as the container would probably do, maybe changing `line-height`). Same for badges on small screens.
- [ ] https://base-apps.github.io/angular-base-apps/#!/icon
  Worth considering adding [Foundation Icon Fonts 3](http://zurb.com/playground/foundation-icon-fonts-3)?

**Comment by soumak77** [comment]
@Seb- Thanks for the amazing feedback!  I fixed most of the issues with the docs and created other issues to track your other suggestions.  Here are my responses to the other (unchecked) suggestions:

#### Migration Tool

This is a great idea but I won't really have the time to create it.  You're welcome to create a new issue for this and hopefully someone from the community will pick it up.

#### Hover Styling

The SCSS code used to style the `:hover` and `:focus` states is shown below:

``` scss
&:hover, &:focus {
  border-color: scale-color($background, $lightness: 25%);
  background: transparent;
  color: scale-color($background, $lightness: 25%);
}
```

The `25%` could be converted into a variable to allow changing the styling.  Feel free to submit a PR or an issue for that feature.

#### Meter & Progress

I'm not all that sure about IE support.  You're probably right it should be moved.  Feel free to submit a PR with those changes.

#### Foundation Icon Fonts 3

This would have to first be supported by [angular-icons](https://github.com/base-apps/angular-icons) and then styling can be added to angular base apps.

**Comment by soumak77** [comment]
I want to finalize the 2.0 release this week and get it published by the end of the weekend.  Let me know if anyone has feedback on the prerelease.

The last thing on the [v2.0 milestone](https://github.com/base-apps/angular-base-apps/milestone/1) is to finalize the template, though that doesn't really need to be completed before v2.0 is published.

**Comment by soumak77** [comment]
Version 2 has been [released](https://github.com/base-apps/angular-base-apps/releases/tag/v2.0.0)
